### PR TITLE
Adding Internationalization (i18n) support to Appwrite console.

### PR DESCRIPTION
**Pull request by goswamianshuman**
## What does this PR do?
- adding i18n support to the console.

## Test Plan
- no code structure was changed.

## Related PRs and Issues
- similar issue PR: https://github.com/appwrite/console/pull/55

### CODE REVIEW

**Comment by goswamianshuman** [comment]
@TorstenDittmann @TGlide please have a look over this PR, the provided i18n is available in German and English.

the main dashboard is not having any changes but you can have a look over the other files.

File: package.json [hunk 1]
```diff
@@ -30,6 +31,7 @@
         "logrocket": "^3.0.1",
         "pretty-bytes": "^6.1.0",
         "prismjs": "^1.29.0",
+        "svelte-i18n": "^3.6.0",
```
Comment (TGlide) [inline package.json:34]: We should discuss if this is the best library for the console. @TorstenDittmann has some features he'd like, and it also seems unmaintained.

File: src/routes/invite/+page.svelte [hunk 2]
```diff
@@ -40,28 +41,29 @@
 </script>
 
 <svelte:head>
-    <title>Accept invite - Appwrite</title>
+    <title>{$_('accept_invite.title')} - Appwrite</title>
```
Comment (TGlide) [inline src/routes/invite/+page.svelte:44]: Out of curiosity, does the `$_` store have autocomplete?

**Comment by goswamianshuman** [comment]
> Really nice start! Not sure if I missed it, is there any way to change the language within the console?

Hey, I will be working on it today. 💪

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
         "node": ">=16"
     },
     "scripts": {
-        "dev": "vite dev",
+        "dev": "vite",
+        "vite": "vite dev",
```
[the remark this answers is not in the document]
Comment (goswamianshuman) [reply package.json:8]: sure, I was testing different libraries so I did the following changes. I'll make it back to the original.

File: src/routes/+layout.svelte [hunk 4]
```diff
@@ -107,6 +107,11 @@
             }
         }
     }
+
+    // $: {
+    //     const getUserLanguage = window.navigator.language;
+    //     console.log(getUserLanguage);
+    // }
```
Comment (TGlide) [inline src/routes/+layout.svelte:114]: This can be removed

File: src/routes/invite/+page.svelte [hunk 5]
```diff
@@ -40,28 +41,29 @@
 </script>
 
 <svelte:head>
-    <title>Accept invite - Appwrite</title>
+    <title>{$_.t('accept_invite.title')} - Appwrite</title>
```
Comment (TGlide) [inline src/routes/invite/+page.svelte:44]: I forgot, is this typesafe? E.g. if I pass in a string that doesn't exist, will the IDE (not the server) return an error?

Comment (goswamianshuman) [reply src/routes/invite/+page.svelte:44]: @TGlide I think now we are working with pull request #459, so we have stopped working on this PR we can close this PR, but first we can have a discussion with Torsten about it first.

**Comment by goswamianshuman** [comment]
Hello, @TorstenDittmann  I think we don't need this PR anymore already we have continued with #459 so I'm closing it for now we can open it again if we require to make any new changes to it.😀

**Pull request closed without merging**